src/fpm_targets.f90

```diff
@@ -191,6 +197,10 @@ subroutine build_target_list(targets,model)
 
             do i=1,size(sources)
 
+                if (.not. include_tests) then
+                    if (sources(i)%unit_scope == FPM_SCOPE_TEST) cycle
+                end if
```

Comment:
It would be beneficial to make the `include_tests` logical part of the `fpm_model_t`, this way it can be easily accessed here without requiring to change the signature of all procedures until we get to the target list builder.

Reply:
I agree that it would be nicer to have it in the model. But it did not seem to fit with the existing, more fundamental components:

```fortran
type :: fpm_model_t
    character(:), allocatable :: package_name
    type(package_t), allocatable :: packages(:)
    type(compiler_t) :: compiler
    type(archiver_t) :: archiver
    character(:), allocatable :: fortran_compile_flags
    character(:), allocatable :: output_directory
    type(string_t), allocatable :: include_dirs(:)
    type(string_t), allocatable :: link_libraries(:)
    type(string_t), allocatable :: external_modules(:)
    type(dependency_tree_t) :: deps

    logical: include_tests
end type fpm_model_t
```
Is this ok?